Dashboard/app/js/lib/views/data/question-answer-view.js

```diff
@@ -349,7 +349,8 @@ FLOW.QuestionAnswerView = Ember.View.extend({
   photoUrl: function(){
     var c = this.content;
     if (!Ember.empty(c.get('value'))) {
-      return FLOW.Env.photo_url_root + c.get('value').split('/').pop();
+      jImage = JSON.parse(c.get('value'));
+      return FLOW.Env.photo_url_root + jImage.filename.split('/').pop();
```

Comment:
it should not happen ideally but perhaps we can add a check here to make sure filename is not `undefined` or `null` etc